evergreen.py

```diff
@@ -99,6 +115,12 @@ def main():  # pragma: no cover
                         title, body, repo, dependabot_file, commit_message
                     )
                     print("\tCreated pull request " + pull.html_url)
+                    if project_id:
+                        pr_id = get_global_pr_id(
+                            token, organization, repo.name, pull.number
+                        )
+                        link_item_to_project(token, project_id, pr_id)
+                        print("\tLinked pull request to project " + project_id)
```

Comment:
Let's only print success if it was in fact successful:
```suggestion
                        response = link_item_to_project(token, project_id, pr_id)
                        if(response):
                            print("\tLinked pull request to project " + project_id)
```

Reply:
Good catch , I agree that's a lot better. The ID that get's printed is the Global ID so does not give to much value.